update/factory/exportHandlerFactory.go

```diff
@@ -253,6 +256,12 @@ func NewExportHandlerFactory(args ArgsExporter) (*exportHandlerFactory, error) {
 
 // Create makes a new export handler
 func (e *exportHandlerFactory) Create() (update.ExportHandler, error) {
+	err := e.prepareFolders(e.exportFolder)
+	if err != nil {
+		log.Error("error while preparing export folder", "error", err)
```

Comment:
This error is not printed again in the caller side?

Reply:
removed. was used for debugging